lib/dbl.php

```diff
@@ -248,7 +254,21 @@ static function parse_connection_params($opt) {
         }
         $cp->host = $cp->host ?? ini_get("mysqli.default_host");
         $cp->user = $cp->user ?? ini_get("mysqli.default_user");
-        $cp->password = $cp->password ?? ini_get("mysqli.default_pw");
+	$cp->password = $cp->password ?? ini_get("mysqli.default_pw");
+
+	if (isset($opt["dbSslKey"]) && is_string($opt["dbSslKey"])) {
```

Comment:
As said above, I'd prefer $opt["dbSsl"] be a boolean, and if false, ignore all the other options.